static/css/misc.less

```diff
@@ -1030,24 +1030,23 @@
     - cfgov-misc
 */
 
+
+// Size of cicular background.
+@size: 1.5em;
```

Comment:
You might give this a more descriptive name, something like `@icon-size__em` maybe?